docs/framework/winforms/windows-forms-accessibility-improvements.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Windows Forms Accessibility Improvements"
+ms.date: "08/16/2019"
+helpviewer_keywords: 
+  - "Windows Forms accessibility"
+  - "accessibility"
+ms.assetid: 
+---
+# Accessibility improvements in Windows Forms controls for .NET Core
+
+## Scope
```

Comment:
Delete lines 11-17; they're based on the compatibility template, which isn't relevant here.

Reply:
Fixed, thanks.